packages/dd-trace/test/plugins/externals.json

```diff
@@ -35,6 +35,10 @@
     {
       "name": "loopback",
       "versions": [">=2.38.1"]
+    },
+    {
+      "name": "cookie-parser",
+      "versions": ["1.4.6"]
```

Comment:
why a fixed version ?

Reply:
No reasons TBH, I saw that some of then was with fixed version, and I put it, but you are right that there are others with `>=` version. I'm changing it.